osu.Game/Beatmaps/Beatmap.cs

```diff
@@ -119,12 +119,11 @@ public double GetMostCommonBeatLength()
         IBeatmap IBeatmap.Clone() => Clone();
 
         public Beatmap<T> Clone() => (Beatmap<T>)MemberwiseClone();
+
+        public override string ToString() => BeatmapInfo.ToString();
```

Comment:
This thing I find slightly disconcerting because there's this but there's also

https://github.com/ppy/osu/blob/9a6857bfadcf7205c166eb2a79df25175c9f5838/osu.Game/Beatmaps/Beatmap.cs#L29-L30

https://github.com/ppy/osu/blob/9a6857bfadcf7205c166eb2a79df25175c9f5838/osu.Game/Beatmaps/Beatmap.cs#L43-L44

What are these other nullchecks? How does r# even infer that this can never be null, on a file with _disabled nullability annotations_ no less?

Reply:
I've enabled NRT and fixed those. It does seem to pass the test gauntlet at least.